src/common/transformations/src/transformations/fix_rt_info.cpp

```diff
@@ -23,17 +22,11 @@ bool ov::pass::FixRtInfo::run_on_model(const std::shared_ptr<ngraph::Function>&
                 run_on_model(sub_graph);
```

Comment:
@ilya-lavrenov u mean this func can be remove? Caused the shared_ptr had been removed, so no need to  extract runtime attributes from shared pointer any more?

Reply:
we discussed with @ilyachur and decided to remove transformation separately. 
So, let's keep it for now